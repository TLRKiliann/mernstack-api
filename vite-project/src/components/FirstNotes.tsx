import { Component } from 'react'
import {notesType} from '../Models/notesType'

export default class FirstNotes extends Component<notesType> {
  constructor(props:any) {
    super(props)
  }
  render() {
    return(
      <div>
        Hello First ! {this.props.name}
      </div>
    )
  }
}