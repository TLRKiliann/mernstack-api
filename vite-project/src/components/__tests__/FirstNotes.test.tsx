//import React from "react";
import {beforeEach, describe, expect, test, it, vi} from 'vitest';
import {fireEvent, render, screen, waitFor} from '@testing-library/react';
//import "@testing-library/jest-dom/extend-expect"
//import "@testing-library/jest-dom";
import { act } from 'react-dom/test-utils';
import {create} from 'react-test-renderer';
import FirstNotes from "../FirstNotes.tsx";

test('MatchSnapShot test FirstNotes 1', () => {
	const tree = create(<FirstNotes />)
	expect(tree.toJSON()).toMatchSnapshot();
})