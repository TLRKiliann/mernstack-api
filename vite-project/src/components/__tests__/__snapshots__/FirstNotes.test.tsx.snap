// Vitest Snapshot v1

exports[`MatchSnapShot test FirstNotes 1 1`] = `
<div>
  Hello First ! 
</div>
`;
